services/app-api/forms/naaar.json

```diff
@@ -197,6 +197,71 @@
               }
             ]
           }
+        },
+        {
+          "name": "Provider type coverage",
+          "path": "/naaar/state-and-program-information/provider-type-coverage",
+          "pageType": "standard",
+          "verbiage": {
+            "intro": {
+              "section": "I. State and program information",
+              "subsection": "Provider type coverage",
+              "info": "Instructions will be here for for how to best classify provider types to the categories shown. If your standards include more specific providers, select the core provider type category closest to the provider type and you can provide it as a subcategory when creating standards later in the form."
```

Comment:
Assuming text is FPO, but there's `for for` here in case it isn't.

Reply:
oh thanks for noticing! I am checking with design about this text because it seems like it's going to change, like you said